src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,49 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    private static final String SEPARATOR = ",|:";
+    private static final String DELIMITER = "//(.)\n(.*)";
+
+    public int add(final String text) {
+        if (text == null || text.isEmpty()) return 0;
```

Comment:
```suggestion
        if (text == null || text.isEmpty()) {
            return 0;
        }
```
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used

구글 스타일 가이드를 참고하여 중괄호를 생략하지 않고 작성해보세요 😄